tests/FakeItEasy.Specs/CreationSpecs.cs

```diff
@@ -123,6 +123,24 @@ public interface ICollectionItem
                 .x(() => fakes.Should().OnlyContain(item => Fake.GetFakeManager(item) != null));
         }
 
+        [Scenario]
+        public void InterfaceWithAlikeGenericMethod()
+        {
+            "Given an interface with methods having multiple generic arguments that can be inheirited"
+                .See<IInterfaceWithSimilarMethods>();
+
+            "It should create fake instance of this"
+                .x(() => this.CreateFake<IInterfaceWithSimilarMethods>());
+
+        }
+
+        public interface IInterfaceWithSimilarMethods
+        {
+            void Test1<T>(IEnumerable<T> enumerable);
+
+            void Test1<T>(IList<T> enumerable);
+        }

```

Comment:
Again a nitpick that I'd only bother with since I'm commenting. The style rules dictate that interfaces appear before methods. Could you move this up next to `ICollectionItem`?

Reply:
Done